CHANGELOG.md

```diff
@@ -7,6 +7,98 @@ documentation before upgrading to a new release.
+
+* Fix some issues involving daylight saving time changes that could result in an hour missing from the time period.
+  This fix applies to downtimes as well. #9238
+
+### Configuration
+
+* Fix the evaluation order of default templates when used in combination with apply rules. Now default templates are
+  imported first as stated in the documentation and as it already happens for objects defined without using apply. #9290
+
+### IDO
+
+* Fix an issue where contacts were not written correctly to the notification history if multiple IDO instances are
+  active on the same node. #9242
+* Explicitly set the encoding for MySQL connections as a workaround for changed defaults in Debian bullseye. #9312
+* Previous versions shipped MySQL schema files with inconsistent version information in the full schema file and upgrade
+  files. This could result in inaccurate reports of an outdated schema version. #9139
+
+### Performance Data Writers
+
+* Fix a race condition in the InfluxDB Writers that could result in a crash. #9237
+* Fix a log message where Influxdb2Writer logged as InfluxdbWriter. #9315
+* Fix an issue in all writers that could result in metrics being written multiple times in non-default HA setups. #9322
+
+### Build
+
+* Fix the order of linker flags to fix builds on some ARM platforms. #9164
+* Fix a regression introduced in 2.13.2 preventing non-unity builds. #9094
+* Fix an issue when building within an unrelated Git repository, version information from that repository could
+  incorrectly be used for Icinga 2. #9155
+* Windows: Update bundled Boost version to 1.78.0 and OpenSSL to 1.1.1n #9325
+
+### Internals
+
+* Fix some race conditions due to missing synchronization. Note: these race conditions should not have caused any
+  practical problems besides incorrect numbers in debug log message. #9306
+* Move the startup.log and status files created when validating incoming cluster config updates to /var/lib/icinga2/api
+  and always keep the last failed startup.log to ease debugging. #9335
+
+### Icinga DB
+
+* The severity attribute was updated to match the sort order Icinga Web 2 uses for the IDO. The documentation for this
+  attribute was already incorrect before and was updated to reflect the current functionality. #9239 #9240
+* Fix the `is_sticky` attribute for comments. #9303
+* Fix missing updates of is_reachable and severity in the state tables. #9241
+* Fix an issue where removing an acknowledgement would write incorrect comment history. #9302
+* Fix multiple issues so that in a HA zone, both nodes now write consistent history. #9157 #9182 #9190
+* Fix an issue where instead of updating the state information, a history event was written. #9252
+* Fix an issue where incomplete comment history events were generated. Note: when removing comments using the API, the
+  dedicated remove-comment action should be used instead of the objects API, otherwise no history event for that action
+  will be generated. #9301
```

Comment:
Consider making the note bold, on a separate Markdown line (not bullet point) and after #PR_NR.